src/transformation.py

```diff
@@ -24,26 +24,32 @@ def execute(self):
             tags = []
         self.prepare_tagged_files(cfg, tags)
 
-        # Process and execute transformation scripts
-        script_file = cfg.get_data_dir() + 'script.py'
-        file = open(script_file, 'w+')
-        self.process_blocks(parameters.get('blocks'), file)
-        file.seek(os.SEEK_SET)
-        self.execute_script_file(cfg, file)
-        file.close()
-
-        # remove temp file
-        os.remove(script_file)
+        blocks = parameters.get('blocks')
+        if blocks is not None:
+            # Process and execute transformation scripts
+            script_file = cfg.get_data_dir() + 'script.py'
+            file = open(script_file, 'w+')
+            self.process_blocks(blocks, file)
+            file.seek(os.SEEK_SET)
+            self.execute_script_file(cfg, file)
+            file.close()
+
+            # remove temp file
+            os.remove(script_file)
 
     def process_blocks(self, blocks, file):
         for block in blocks:
             print('Processing block "%s"' % (block.get('name')))
-            self.process_codes(block.get('codes'), file)
+            codes = block.get('codes')
+            if codes is not None:
+                self.process_codes(codes, file)
 
     def process_codes(self, codes, file):
         for code in codes:
             print('Processing code "%s"' % (code.get('name')))
-            self.process_script(code.get('script'), file)
+            scripts = code.get('script')
+            if scripts is not None:
```

Comment:
Ani tato situacia asi nenastane ... prechadzas pole, ... muselo by to byt pole `[None, None, ...]` co sa tam nestane ... ked bude pole prazdne, tak sa k tomuto kodu nedostanes ... (`None` je ako `null` v PHP, takze ma zmysel iba v tej prvej podmienke, ked citas konfiguraciu a kluc tam nie je).

Reply:
jsi si jistý že to nemůže nastat, tohl kontroluje pole 
`["script": None]` tj že tam není vložený žádný script

Reply:
já ty 2 podmínky můžu dát pryč, ale zazačátku jsem měl za to že to nemůže nstat ani v tom prvním případě :)

Reply:
proto jsem to dal pro jistotu všude a imho to tam ničemu nevadí

Reply:
... no to nastalo, lebo to umoznilo UI spustit prazdny config, mali by sme to vediet povedat jednoznacne, alebo sa opytat UI teamu ak mas pochybnosti, ... lebo potom musime opravit aj ostatne transformacie ...

Aby bolo `["script": None]` , ... musela byt tam byt hodnota `null` v JSONe ... ked tam moze byt `null`, tak podme aj ostatne transformacie opravit (ako som uz pisal), ... ak nie, tak to nedava zmysel ani tu ... ako kludne ti dam approve, ... len velmi nie som zastanca toho, ze "nie som si isty ci to tam treba, tak to tam radsej dam", lepsie je to podla mna vyriesit jednoznacne (ak sa da).

Reply:
no ono u ostatních transformací nemůže ani `blocks` být `null`... např.
https://github.com/keboola/snowflake-transformation/blob/2ee7dbf2910c11fe98e9dbf5828e272e514d23f3/src/Config.php#L17

Reply:
takze tam to tiez spadne ked spustis prazdny config, ... jedine ze by to bolo na UI inak implementovane.

Reply:
alebo to mozno osetri symfony config builder, ... ze vrati prazdne pole (bolo by to potrebne vyskusat).

Reply:
https://keboola.slack.com/archives/CMTBFNLF8/p1604588623237900

Reply:
teď jsem se k tomu konečně dostal - ty ostatní kontroly tam nejsou potřeba - dal jsem je pryč

Reply:
@webrouse koukneš na to ještě jednou psl?